docs/data/datamodel.md

```diff
@@ -1,8 +1,333 @@
+        string shortcode "1"
+        string status "1; Literal 'Ongoing' or 'Finished'"
+        string name "1"
+        lang_string description "1"
+        date startDate "1"
+        string teaserText "1"
+        url url "1"
+        string howToCite "1"
+        id[] datasets "1-n; Dataset IDs"
+        lang_string[] keywords "1-n"
+        lang_string_or_url[] disciplines "1-n"
+        lang_string_or_url[] temporalCoverage "1-n"
+        url[] spatialCoverage "1-n"
+        id[] funders "1-n; Person or Organization IDs"
+        date endDate "0-1"
+        url secondaryURL "0-1"
+        dmp dataManagementPlan "0-1"
+        id contactPoint "0-1"
+        publication[] publications "0-n"
+        id[] grants "0-n"
+        lang_string[] alternativeNames "0-n"
+    }
+
+    dataset {
+        string __id "1"
+        string __type "1; Literal 'Dataset'"
+        string title "1"
+        string accessConditions "1; Literal 'open', 'restricted' or 'closed'"
+        string howToCite "1"
+        string status "1; Literal 'In Planning', 'Ongoing', 'On hold', 'Finished'"
+        lang_string_or_url[] abstract "1-n"
+        string[] typeOfData "1-n; Literal 'XML', 'Text', 'Image', 'Video', 'Audio'"
+        license[] licenses "1-n"
+        lang_string[] languages "1-n"
+        attribution[] attributions "1-n"
+        date datePublished "0-1"
+        date dateCreated "0-1"
+        date dateModified "0-1"
+        url distribution "0-1"
+        lang_string[] alternativeTitles "0-n"
+        url[] urls "0-n"
+        lang_string_or_url[] additional "0-n"
+    }
+
+    person {
+        string __id "1"
+        string __type "1; Literal 'Person'"
+        string[] givenNames "1-n"
+        string[] familyNames "1-n"
+        string[] jobTitles "0-n"
+        id[] affiliations "0-n; Organization IDs"
+        address address "0-1"
+        string email "0-1"
+        string secondaryEmail "0-1"
+        url[] authorityRefs "0-n"
+    }
+
+    organization {
+        string __id "1"
+        string __type "1; Literal 'Organization'"
+        string name "1"
+        url url "1"
+        address address "0-1"
+        string email "0-1"
+        lang_string alternativeName "0-1"
+        url[] authorityRefs "0-n"
+    }
+
+    grant {
+        string __id "1"
+        string __type "1; Literal 'Grant'"
+        id[] funders "1-n; Person or Organization IDs"
+        string number "0-1"
+        string name "0-1"
+        url url "0-1"
+    }
+```
+
+
+## Legacy docs
+
+The legacy documentation can be found [here](https://github.com/dasch-swiss/dsp-meta-svc/tree/main/docs/services/metadata).
+This includes thoughts on mappings to external aggregators 
+and on RDF representation of the metadata.
+
+
+## Issues/Questions
+
+- access conditions on datasets not current state
+- status on datasets not aligned with project status
+- typeOfData on datasets is a weird list of options
+- does dataset need all the date fields?
```

Comment:
```suggestion
- access conditions on datasets not current state,
- status on datasets not aligned with project status,
- typeOfData on datasets is a weird list of options,
- does dataset need all the date fields?
```

Reply:
I actually meant to remove this entirely, before merging. Thanks, would otherwise have forgotten about this.